src/pages/PageWrapper.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
Can we put this in a separate folder from pages?

IMO src/pages/.* should be more focused on composing components and business logic. This component covers more styling. We can put this in components or create a new folder